prac_05/emails.py

```diff
@@ -0,0 +1,39 @@
+"""
+CP1404/CP5632 Practical
+Emails
+Estimate: 40 minutes
+Actual:   45 minutes
+"""
+
```

Comment:
This part is almost perfect, only if you can add some more comments or docstring to clarify your process, it would be perfect !